internal/store/requirements.go

```
@@ -177,6 +179,94 @@ func (m *memory) ListRequirements(ctx context.Context) ([]core.Requirement, erro
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	workspace := workspaceOrDefault(ctx, "")
+	key := memoryScopedKey{workspace: workspace, id: id}
+	document, ok := m.requirements[key]
+	if !ok {
+		return fmt.Errorf("%w: requirement %s", ErrNotFound, id)
+	}
+	if document.Archived == archived {
+		return nil
+	}
+	accepted := []string{}
```

golangci-lint (2.13.2)

[error] ineffectual assignment to accepted

(ineffassign)